app/commands/v2/discard_draft.rb

```diff
@@ -13,8 +13,18 @@ def call
         increment_live_lock_version if live
 
         after_transaction_commit do
-          delete_draft_from_draft_content_store(draft_path)
-          send_live_to_draft_content_store(live)
+          if live
+            live_path = Location.find_by!(content_item: live).base_path
+
+            if live_path == draft_path
```

Comment:
Could we simplify this to make it a bit easier to follow?

Maybe

```
send_live_to_draft_content_store(live)
delete_draft_from_draft_content_store(draft_path) if live_path != draft_path
```
or even extract `live_path == draft_path` into a method named to describe what that actually means?